### PR TITLE
Introduce a conda-forge recipe for PyOmicron

### PR DESCRIPTION
**Pull request by alurban**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by alurban** [comment]
@duncanmmacleod has expressed consent to me privately, but may want to leave a comment on this pull request confirming.

@conda-forge/help-python, the source code for this package is written entirely in python, but it is ineligible for `noarch: python` because its `setup.py` file utilizes the `scripts` argument (i.e., python scripts are included in a `bin` directory intended to be used like binary executables rather than python modules).

The package may also build successfully on all platforms, but we do not expect all dependencies to build on Windows.

File: recipes/pyomicron/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    - htcondor
+    - ligo-segments
+    - lscsoft-glue >=1.60.0
+    - markuppy
+    - numpy
+    - omicron >=2.3.12
+    - pycondor
+    - python-ldas-tools-framecpp
+    - python-ligo-lw >=1.4.0
+    - root_base 6.18.04
```
[the remark this answers is not in the document]
Comment (alurban) [reply recipes/pyomicron/meta.yaml:36]: Ah thanks for catching this, I've fixed it in my local clone and will push shortly.

**Comment by alurban** [comment]
@conda-forge/help-python, this PR is now mature and ready for review

File: recipes/pyomicron/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f13c7d758ce2ec75acf9940d9fba4590807e0cbe96fc997f5b21b1b30eacac17"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  skip: True  # [not linux]
```
Comment (scopatz) [inline recipes/pyomicron/meta.yaml:15]: This can be a noarch: python recipe

**Comment by alurban** [comment]
@scopatz, I had tried that, but the windows and macOS builds each failed due to missing dependencies on those systems. With noarch: python packages, what is the correct way to skip non-linux builds?

**Comment by scopatz** [comment]
noarch python recipes only need to build on linux. It is ok that they failed on OSX and Win

**Comment by alurban** [comment]
Thanks @scopatz, I hadn't appreciated that, I've reverted back to `python: noarch` and squashed my commits

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**